packages/blob/src/client.browser.test.ts

```diff
@@ -133,9 +133,12 @@ describe('client', () => {
       const pathname = 'foo.txt';
       const token = 'vercel_blob_client_fake_token_for_test';
 
+      const abortController = new AbortController();
+
       const { uploadId, key } = await createMultipartUpload(pathname, {
         access: 'public',
         token,
+        abortSignal: abortController.signal,
```

Comment:
Do we need this for the tests? I am confused as it seems we added an abortSignal to all the tests.

Reply:
👍 got it now!